app/(frontend)/(chat)/chatbox.tsx

```
@@ -55,122 +71,98 @@ export default function ChatBox() {
-    // ("Schema API response:", responseData);
-
-    const schemas = Array.isArray(responseData)
-      ? responseData
-      : responseData.data;
-
-    if (!Array.isArray(schemas)) {
+    // Handle different response formats more explicitly
+    let schemas = [];
+    if (Array.isArray(responseData)) {
+      schemas = responseData;
+    } else if (responseData && responseData.data && Array.isArray(responseData.data)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)